deploy/monitor/push_prometheus_rule.yaml

```diff
@@ -14,78 +14,33 @@ spec:
   groups:
     - name: general.rules
       rules:
-      - alert: UnifiedpushDown
-        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="ups"}==1)
+      - alert: UnifiedPushDown
+        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="ups"} < 1)
         for: 5m
         labels:
           severity: critical
         annotations:
           description: "The UnifiedPush Pod Server has been down for more than 5 minutes. "
           summary: "The UnifiedPush Pod Server is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-server"
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc"
       - alert: UnifiedPushConsoleDown
-        expr: absent(kube_endpoint_address_available{endpoint="example-unifiedpushserver-unifiedpush"} == 1)
+        expr: absent(kube_endpoint_address_available{endpoint="example-unifiedpushserver-unifiedpush"} < 1)
         for: 5m
         labels:
           severity: critical
         annotations:
           description: "The UnifiedPush admin console has been down for more than 5 minutes. "
           summary: "The UnifiedPush admin console endpoint has been unavailable for more that 5 minutes. For more information see on the UnifiedPush  at https://github.com/aerogear/UnifiedPush "
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc"
       - alert: UnifiedPushDatabaseDown
-        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="postgresql"}==1)
+        expr: absent(kube_pod_container_status_running{namespace="unifiedpush",container="postgresql"} < 1)
         for: 5m
         labels: 
           severity: critical
         annotations:
           description: "The UnifiedPush Database pod has been down for more than 5 minutes"
           summary: "The UnifiedPush Database is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-serve"
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
-      - alert: UnifiedPushPodCPUHigh # todo check if it will work after expse the metrcis
-        expr: "(rate(process_cpu_seconds_total{job='example-unifiedpushserver-unifiedpush'}[1m])) > (((kube_pod_container_resource_limits_cpu_cores{namespace='mobile-security-service',container='application'})/100)*90)"
```

Comment:
@wei-lee and @grdryn we are not able to get the `kube_pod_container_resource_limits_cpu_cores{namespace='unifiedpush'}` for the ups and database because these containers are not configured with limits as it is done here for an example: https://github.com/aerogear/mobile-security-service-operator/blob/master/pkg/controller/mobilesecurityservice/deployments.go#L119. So we are not able to create alerts over it. IMHO: We can move forward with but would be great track this need.